src/fuse_pointwise.cpp

```diff
@@ -204,12 +205,17 @@ struct pointwise_reshape : rewrite_reshapes_base
 void fuse_pointwise::apply(module_pass_manager& mpm) const
 {
     mpm.run_pass(eliminate_identity{});
-    create_pointwise_modules(mpm);
-    mpm.run_pass(dead_code_elimination{});
-    if(enabled(MIGRAPHX_DISABLE_POINTWISE_FUSION{}))
+    size_t pointwise_cnt = 0;
+    if(disable_fusion)
     {
+        create_pointwise_modules(mpm, pointwise_cnt, {"layout"});
+        mpm.run_pass(dead_code_elimination{});
         return;
     }
+    // Excluding `logical_and` and `where` from fusion
+    // to fix GPU accuarcy of GridSample: https://github.com/ROCm/AMDMIGraphX/issues/2923
+    create_pointwise_modules(mpm, pointwise_cnt, {"layout", "logical_and", "where"});
```

Comment:
This has major perf problems. I dont think we should be skipping fusions. Instead we should fix the GPU kernel.

Reply:
@pfultz2 GPU kernel debuging is unknown area for me. Can you give me som pointers regaridng how can I start debugging this?

Reply:
I added some comments to the issue.